AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSCosmosDb.m

```diff
@@ -128,42 +128,41 @@ + (NSString *)documentUrlWithTokenResult:(MSTokenResult *)tokenResult documentId
   return [MSCosmosDb documentDbEndpointWithDbAccount:tokenResult.dbAccount documentResourceId:documentResourceIdPrefix];
 }
 
-+ (void)performCosmosDbAsyncOperationWithHttpClient:(MSCosmosDbIngestion *)httpClient
++ (void)performCosmosDbAsyncOperationWithHttpClient:(id<MSHttpClientProtocol>)httpClient
                                         tokenResult:(MSTokenResult *)tokenResult
                                          documentId:(NSString *)documentId
                                          httpMethod:(NSString *)httpMethod
                                                body:(nullable NSData *)body
-                                  completionHandler:(MSCosmosDbCompletionHandler)completionHandler {
-
+                                  completionHandler:(MSHttpRequestCompletionHandler)completionHandler {
   [MSCosmosDb performCosmosDbAsyncOperationWithHttpClient:httpClient
                                               tokenResult:tokenResult
                                                documentId:documentId
                                                httpMethod:httpMethod
                                                      body:body
                                         additionalHeaders:nil
+                                              offlineMode:NO
```

Comment:
This shouldn't be `NO`. However the method is a dead code and only be used in test. Probably you can delete the method.

Reply:
If you don't delete the method, you should add `offlineMode` to the method parameter.